README.md

```diff
@@ -115,362 +115,104 @@ global_config.set_default(
 
 ## Usage
 
-### Create client
+Please find more usages for this sdk at [here](https://www.instill.tech/docs/sdk/python#usage)
 
 **You can also find some notebook examples [here](https://github.com/instill-ai/python-sdk/tree/main/notebooks)**
 
-Simply import the `get_client` function to get the client that are connected to all services with the config you setup previously.
+### Create a model
 
-```python
-from instill.clients import get_client
-
-client = get_client()
-```
-
-> [!NOTE]  
-> Remember to call `client.close()` at the end of script to release the channel and the underlying resources
-
-If you have not set up `Instill VDP` or `Instill Model`, you will get a warning like this:
-
-```bash
-2023-09-27 18:49:04,871.871 WARNING  Instill VDP is not serving, VDP functionalities will not work
-2023-09-27 18:49:04,907.907 WARNING  Instill Model is not serving, Model functionalities will not work
-```
-
-You can check the readiness of each service:
-
-```python
-client.mgmt_service.is_serving()
-# True
-client.pipeline_service.is_serving()
-# True
-client.model_service.is_serving()
-# True
-```
-
-You can also switch to other instances
+Now create a model `text-generation` in `Instill Model` for later use
 
 ```python
-client.set_instance("your-instance-in-config")
-client.mgmt_service.instance
-# 'your-instance-in-config'
-```
-
-After making sure all desired services are serving, we can check the user status by:
-
-```python
-client.mgmt_service.get_user()
-```
-
-If you have a valid `api_token` in your config file, you should see something like this:
-
-```
-name: "users/admin"
-uid: "4767b74d-640a-4cdf-9c6d-7bb0e36098a0"
-id: "admin"
-type: OWNER_TYPE_USER
-create_time {
-  seconds: 1695589596
-  nanos: 36522000
-}
-update_time {
-  seconds: 1695589749
-  nanos: 544980000
-}
-email: "hello@instill.tech"
-first_name: "Instill"
-last_name: "AI"
-org_name: "Instill AI"
-role: "hobbyist"
-newsletter_subscription: true
-cookie_token: ""
-```
-
-#### Now we can proceed to create resources
-
-### Create Model
-
-Let's say we want to serve a `yolov7` model from `github` with the following configs
-
-```python
-model_name = "yolov7"
-model_repo = "instill-ai/model-yolov7-dvc"
-model_tag = "v1.0-cpu"
-```
-
-Simply import the GithubModel resource and fill in the corresponding fields
-
-```python
-from instill.resources.model import GithubModel
-
-yolov7 = GithubModel(
-  client=client,
-  name=model_name,
-  model_repo=model_repo,
-  model_tag=model_tag,
+import instill.protogen.common.task.v1alpha.task_pb2 as task_interface
+model_id = "model_text-generation"
+client.model_service.create_model(
+    model_id,
+    task_interface.Task.TASK_TEXT_GENERATION,
+    "REGION_GCP_EUROPE_WEST4",
+    "CPU",
+    "model-definitions/container",
+    {},
```

Comment:
It's weird to have this part before the introduction of `client`

Reply:
fixed